selected_polygon.geojson

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
In time  this is the sort of thing I could see moved to the tests assuming we are making a generic tool for creating DigitalTwins that is independent of a single location. 

Definitely not worth working about now though! And good work simplifying/bringing dependencies into the fold.

Reply:
Hi @rosepearson, yes I agree. For now this is the first step of simplifying it and soon we will replace this with a polygon submitted by a user. Baby steps until then